<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="18997" started-at="2017-01-10T11:48:47Z" finished-at="2017-01-10T11:49:06Z">
    <groups>
    </groups>
    <test name="IE Test" duration-ms="18997" started-at="2017-01-10T11:48:47Z" finished-at="2017-01-10T11:49:06Z">
      <class name="RACFtoAD.Applications">
        <test-method status="PASS" signature="beforeTest(java.lang.String)[pri:0, instance:RACFtoAD.Applications@675d3402]" name="beforeTest" is-config="true" duration-ms="17895" started-at="2017-01-10T17:18:47Z" finished-at="2017-01-10T17:19:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[IE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="Add_Application()[pri:0, instance:RACFtoAD.Applications@675d3402]" name="Add_Application" duration-ms="64" started-at="2017-01-10T17:19:04Z" finished-at="2017-01-10T17:19:05Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at RACFtoAD.Helper.CreateWebElement(Helper.java:192)
	at RACFtoAD.Helper.NavigateTo(Helper.java:107)
	at RACFtoAD.Helper.AddApplication(Helper.java:864)
	at RACFtoAD.Applications.Add_Application(Applications.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_Application -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:RACFtoAD.Applications@675d3402]" name="afterTest" is-config="true" duration-ms="1019" started-at="2017-01-10T17:19:05Z" finished-at="2017-01-10T17:19:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- RACFtoAD.Applications -->
    </test> <!-- IE Test -->
  </suite> <!-- Suite -->
</testng-results>
